cmd/helm/install.go

```diff
@@ -235,12 +236,24 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		if err := action.CheckDependencies(chartRequested, req); err != nil {
 			err = errors.Wrap(err, "An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies")
 			if client.DependencyUpdate {
+				// there is no way to get a ready-made registryClient except  creating a new one.
```

Comment:
it seems `client *action.Install`  holds a reference to a registryClient through (private) cfg field: 
https://github.com/helm/helm/blob/main/pkg/action/install.go#L67 , also accessible with (again private) ` ChartPathOptions.registryClient` and 
initialized here : https://github.com/helm/helm/blob/main/cmd/helm/root.go#164,

I guess we should make it accessible from client and reuse instead of creating a new one